layout.js

```diff
@@ -10,20 +10,22 @@ const slideLayoutStyle = css`
   h4,h5,h6 {
     color: #ffffff;
```

Comment:
here I added the color for small headings as white, but perhaps better to have all headings as taxfix green. And leave the default for text as white.